R/foundInSlurm.R

```diff
@@ -11,11 +11,7 @@
 foundInSlurm <- function(mydir = ".", user = NULL) {
 
   if (is.null(user)) user <- Sys.info()[["user"]]
```

Comment:
the exception for coupled runs is not necessary anymore, as their job name is equal to the folder name now.